cavert-pkirch/pointers/wallet.go

```diff
@@ -0,0 +1,40 @@
+package pointers
+
+import (
+	"errors"
+	"fmt"
+)
+
+type Bitcoin int
+
+type Wallet struct {
```

Comment:
This is pure purism. Simple. Beautiful. No comment and documentation required ;)

Reply:
> Beautiful code. LGTM.

I had to google that :-D